src/FffGcodeWriter.cpp

```diff
@@ -827,6 +827,33 @@ void FffGcodeWriter::addMeshLayerToGCode(SliceDataStorage& storage, SliceMeshSto
     }
     part_order_optimizer.optimize();
 
+    if (mesh->infill_angles.size() == 0)
+    {
+        mesh->infill_angles = mesh->getSettingAsIntegerList("infill_angles");
```

Comment:
In general angles are provided in the frontend as floating point numbers.
I don't really consider this as a blocking issue, but it would be more neat if the list of angles is a list of floats.